bugsnag-spring/build.gradle

```diff
@@ -0,0 +1,23 @@
+apply plugin: 'java-library'
+apply plugin: 'checkstyle'
+
+sourceCompatibility = 1.6
+targetCompatibility = 1.6
+
+repositories {
+    jcenter()
+}
+
+dependencies {
+}
+
+test {
+    testLogging {
+        events "passed", "skipped", "failed", "standardOut", "standardError"
+    }
+}
+
+checkstyle {
+    toolVersion = "6.16"
+    configFile = new File(rootDir, "config/checkstyle/checkstyle.xml")
+}
```

Comment:
It may be worth extracting some of this configuration to a [separate script](https://docs.gradle.org/current/userguide/plugins.html#sec:script_plugins), then applying it to both modules. That way it would only be necessary to update checkstyle/test/source compat in one place.